dweather_client/client.py

```diff
@@ -139,15 +146,30 @@ def get_forecast(
         raise DatasetError("No such dataset in dClimate")
 
     # set up units
-    if not desired_units:
-        converter, dweather_unit = get_unit_converter(metadata["unit of measurement"], use_imperial_units)
+    metadata_unit = metadata["unit of measurement"]
+    if type(metadata_unit) == list:
+        metadata_unit = metadata_unit[0]
+
+    if not desired_units and not use_imperial_units:
+        converter = None
+        dweather_unit = u.Unit(metadata_unit)
```

Comment:
tests are failing because of this line. we need to apply aliases, since percentage is not a valid unit by default (the mapping in `UNIT_ALIASES` fixes this)